apps/wallet-mobile/src/features/ReviewTx/useCases/ReviewTxScreen/Overview/OverviewTab.tsx

```diff
@@ -0,0 +1,355 @@
+// 🚧 TODO: grouping by staking address 🚧
```

Comment:
Yes, because multi-receiver has not been implemented yet. And when it is implemented, it will be required to group outputs by staking key

A transaction can be created to several addresses and some of them belong to the same wallet

Reply:
For now I'm just covering our send funnel